src/test/java/harness/utils/ExternalBQDatasets.java

```diff
@@ -0,0 +1,69 @@
+package harness.utils;
+
+import static harness.TestExternalResources.getProjectId;
+import static harness.TestExternalResources.getSACredentials;
```

Comment:
optional style nit: I don't think these method names are so unambiguous that staticly importing them helps with readability. `TestExternalResources.getProjectId()` I think improves readability in this file.
Ditto on external buckets file.

Reply:
Removed static imports